### DISCUSSION TITLE
Netbox 3.0-Beta 1 Home page issue

### DISCUSSION THREAD

**Discussion by QuinnMatthews** [netbox-community/netbox#6598] [Q&A] [answered]
This may be a bug but I wanted to see if anyone else has the same issue. 

I recently upgraded a lab netbox install to the beta and after doing so the home page shows a lock icon next to all of the selections even if a user has permissions for those sections. 

If I go to the tabs on the side then it lets me edit it with no problems. 

It does use LDAP for authentication but the issue occurs even with local users.

**Comment by candlerb** [comment, marked as answer]
I wasn't aware of any netbox 3.0 beta release yet.  I can't find it via a google search, and there is no 3.x tag in the netbox github repository.  Can you provide a link to what code you are running?

Aside: you may already be aware of this, but if you install any development release, it *cannot* be updated to a future official release - and may not even update to a future development release.  So make sure you're only doing this on dummy data, or an export of data from your production system (which you keep on a mainline release).

**Reply by QuinnMatthews** [reply]
It's the labeled as the feature branch here: https://github.com/netbox-community/netbox/tree/feature (Which may not officially be considered a 3.0 release yet? But that is what it is labeled as at the bottom of the screen)

I wasn't aware of that but it is lab data so not a big deal if it has to be wiped. Just out of curiosity is that because of the database migrations or something else?

Of course any production environment would be running official mainline releases.

**Reply by jeremystretch** [reply]
There has been no beta release for v3.0 yet. The `feature` and `develop` branches are for development only. Please don't use these unless you are actively working on modifying NetBox code under the context of a feature request or bug report.

**Comment by candlerb** [comment]
Yep, it's the database migrations.
